frontend/src/static/js/components/webstatus-columns-dialog.ts

```diff
@@ -58,41 +68,70 @@ export class WebstatusColumnsDialog extends LitElement {
 
   handleSave() {
     const newColumns: string[] = [];
+    const columnOptions: string[] = [];
     this.shadowRoot!.querySelectorAll('sl-checkbox').forEach(cb => {
+      if (cb.classList.contains('column-option') && !cb.checked) {
+        columnOptions.push(cb.value);
+      }
       if (cb.checked) {
         newColumns.push(cb.value);
       }
     });
     this.hide();
-    const nextUrl = this.formatUrlWithColumns(newColumns);
+    const nextUrl = this.formatUrlWithColumns(newColumns, columnOptions);
     window.location.href = nextUrl;
   }
 
-  formatUrlWithColumns(columns: string[]): string {
-    return formatOverviewPageUrl(this.location, {columns});
+  formatUrlWithColumns(columns: string[], columnOptions: string[]): string {
+    return formatOverviewPageUrl(this.location, {
+      columns,
+      column_options: columnOptions,
+    });
   }
 
   renderDialogContent(): TemplateResult {
     if (!this.location) return html``;
     const columns: ColumnKey[] = parseColumnsSpec(
       getColumnsSpec(this.location)
     );
+    const columnOptions: ColumnOptionKey[] = parseColumnOptions(
+      getColumnOptions(this.location)
+    );
     const checkboxes: TemplateResult[] = [];
     for (const enumKeyStr of Object.keys(ColumnKey)) {
       const ck = enumKeyStr as keyof typeof ColumnKey;
       const columnId = ColumnKey[ck];
       const displayName = CELL_DEFS[columnId].nameInDialog;
-      checkboxes.push(html`
-        <sl-checkbox
-          value="${columnId}"
-          ?checked=${columns.includes(ColumnKey[ck])}
-        >
-          ${displayName}
-        </sl-checkbox>
-      `);
+      const cellColumnOptions = CELL_DEFS[columnId].options.columnOptions;
+      const checkbox = html`
+        <sl-tree-item expanded>
+          <sl-checkbox
```

Comment:
Add a class here for `column`. This will be separate from the other checkbox from with `column-option`